src/js/tracing/nativeframes.ts

```diff
@@ -150,12 +150,15 @@ export class NativeFramesInstrumentation {
     const measurements = {
       frames_total: {
         value: finalFinishFrames.totalFrames - startFrames.totalFrames,
+        unit: ''
```

Comment:
Can we just ignore them or its mandatory now?

Reply:
yes, the unit is now required:
https://github.com/getsentry/sentry-javascript/blob/838dcfb8808b9b2a851b08dd4e942e48cbb40e50/packages/types/src/transaction.ts#L127